### DISCUSSION TITLE
CEMU Not Recognizing ToyPad EMulator

### DISCUSSION THREAD

**Discussion by Szoszenka** [Berny23/LD-ToyPad-Emulator#38] [Q&A]
![image_2021-12-12_191008](https://user-images.githubusercontent.com/96003537/145724233-9517bbda-4324-4647-931f-e8717335c3af.png)
![image_2021-12-12_191026](https://user-images.githubusercontent.com/96003537/145724246-6dc67505-975f-41dc-ae7f-dc5d32620ca1.png)
Does anyone know how to fix this? VirtualHere is open and connected sucessfully but the game doesnt recognize the ToyPad

**Comment by wiktorzm** [comment]
Try opening http://debian on Windows, not in VM (also make sure the toypad is launched before running cemu)

**Reply by Berny23** [reply]
Yes, running the toy pad emulator before starting Cemu is necessary. This is a limitation by the Cemu developers, so I can't do anything about it.
